crypto/data_formats_test.go

```diff
@@ -15,11 +15,26 @@
 package crypto
 
 import (
+	"encoding/hex"
 	"testing"
 
 	"github.com/google/trillian"
 )
 
+// It's important that signatures don't change.
+var expectedSigHex = "5e6baba8dc3465de9c01d669059dda590b7ce123d6ccd436bcd898f1c79ff6d9"
```

Comment:
nit: Move inside the test? It's not used anywhere else.

Reply:
It looked quite ugly.

Reply:
Oh move the var, sorry. Misunderstood.

Reply:
Done.